pkg/webhooks/v1beta1_suite_test.go

```diff
@@ -39,6 +40,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 )
 
+const alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
just out of curiosity, are we limited by an English alphabet here?

Reply:
The validation on the field I had down as `[a-zA-Z]+`, do you think that is too restrictive? The proposal suggests a CamelCase string and something easily computer readable, hence sticking to basic ascii.